tests/integration/jit/function/skipif.inc

```diff
@@ -0,0 +1,10 @@
+<?php
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+if (version_compare(PHP_VERSION, "8.0", "<")) {
+  die("skip: PHP < 8.0.0 not supported\n");
+}
```

Comment:
```suggestion
}
if (!extension_loaded('Zend OPcache')) {
  die("skip: Zend OPcache extension required\n");
}
```
These tests require `Zend OPcache` extension to be available and loaded. Only then Zend engine is capable of detecting incompatibility and issuing this warning:
```
Warning: JIT is incompatible with third party extensions that override zend_execute_ex(). JIT disabled. in Unknown on line 0
```
if it detects that `newrelic.so` under test is not using Observer API to hook into PHP. Only this warning will cause the tests to fail. Otherwise all tests pass regardless if instrumentation is done via `zend_execute_ex()` hook override or via Observer API. I think these tests should prove that the agent is compatible with Zend's OPcache JIT and it's best if they fail when they're executed with an agent without Observer API or Zend detects other incompatibility.

Reply:
I made this change in commit [b2a47c](https://github.com/newrelic/newrelic-php-agent/pull/583/commits/b2a47cf5a9fec76c1557d24f184983559db66fef).

After ensuring ```opcache.so``` was loaded the following tests now fail (pass w/o opcache):

tests/integration/jit/tracing/test_span_class_function.php
tests/integration/jit/function/test_span_events_are_created_upon_caught_exception.php